core/src/cats/drop_bareos_database.in

```diff
@@ -3,7 +3,7 @@
 # BAREOS® - Backup Archiving REcovery Open Sourced
 #
 # Copyright (C) 2000-2011 Free Software Foundation Europe e.V.
-# Copyright (C) 2013-2023 Bareos GmbH & Co. KG
+# Copyright (C) 2013-2024 Bareos GmbH & Co. KG
```

Comment:
unneeded now

Reply:
I don't understand as the first commit --if-exists change the file and was in 2024 ?

Reply:
You are right. Github had only selected one of the two commits ...